cluster/images/etcd/migrate/options.go

```diff
@@ -115,7 +117,7 @@ func fallbackToEnv(flag, env string) (string, error) {
 	return lookupEnv(env)
 }
 
-func fallbackToEnvWithDefault(flag, env, def string) string {
+func fallbackToEnvWithDefault(env, def string) string {
```

Comment:
Why do you change this? It seems completely unrelated to this PR, so it should not be here.
If you really need it, please open a separate PR, but I actually think it's not a desired change.

Reply:
Reverted, added flag to logging line for better static analysis result.